spec/features/course/lesson_plan_spec.rb

```diff
@@ -32,6 +32,9 @@
     end
 
     before do
+      course.settings.course_lesson_plan_component = { milestones_expanded: 'all' }
```

Comment:
Would it be better to put this in a factory trait instead?

Reply:
I chose not to do that because I thought it was very specific to this page and was very likely to be used elsewhere. Didn't think it was worth the extra level of indirection + clutter in factory file, but I don't think it's a problem to put it there also.